test/signin.test.js

```diff
@@ -0,0 +1,124 @@
+const request = require('supertest'),
+  app = require('../app.js'),
+  validationErrorStatus = 401,
+  firstName = 'fn1',
+  lastName = 'ln1',
+  correctPassword = 'password',
+  correctEmail = 'email@wolox.com.ar',
+  correctEmail1 = 'email2@wolox.com.ar';
+
+describe('POST /susers/sessions', () => {
+  describe('Test missing parameters', () => {
+    test.each([
+      {
+        email: correctEmail
+      },
+      {
+        password: correctPassword
+      },
+      {}
+    ])('Test missing parameter with body = %p', body =>
+      request(app)
+        .post('/users/sessions')
+        .send(body)
+        .then(response => {
+          expect(response.status).toBe(validationErrorStatus);
+        })
+    );
+  });
+
+  describe('Test invalid password', () => {
+    test('Sign in with short password', () =>
+      request(app)
+        .post('/users/sessions')
+        .send({
+          email: correctEmail,
+          password: 'pass'
+        })
+        .then(response => {
+          expect(response.status).toBe(validationErrorStatus);
+        }));
+
+    test('Sign in with password with invalid characters', () =>
+      request(app)
+        .post('/users/sessions')
+        .send({
+          email: correctEmail,
+          password: 'passwo..rd'
+        })
+        .then(response => {
+          expect(response.status).toBe(validationErrorStatus);
+        }));
+
+    test('Valid user but not matching password', () => {
+      request(app)
+        .post('/users')
+        .send({
+          first_name: firstName,
+          last_name: lastName,
```

Comment:
Copy pasta from signIn tests? These two params are not part of this endpoint.

As a side note:
Usually tests are low priority as far as code quality goes, but if you'd like you can try experimenting a bit with ways to make the tests reusable, to avoid copying these tests to every endpoint that has these validations.
There are several options that I can think of, but see if you can think of another:
1. Make unit tests for the validation middleware itself instead of the controller.
2. Pass the endpoint to a describe.each and put these validation tests inside said describe. It would screw with the describe structure a bit, but I feel its wroth it.
3. Define these tests inside a describe in another file and then import said describe. Js does not usually handle imports of tests well but it can be worth for you to try and see the limitations for yourself.
Again, this is optional, so don't get lost in trying to get it perfect. It's fine if you don't finish it. Progressing through the cards is more important.

Also, as another side note, we usually have one test file per controller, instead of per functionality. If the controller grows too large (500 + tests) then we split. Not necesary that you change this however, just a headsup.